lib/linters/localized_validation_message_linter.rb

```diff
@@ -0,0 +1,40 @@
+module RuboCop
+  module Cop
+    module IdentityIdp
+      class LocalizedValidationMessageLinter < RuboCop::Cop::Cop
```

Comment:
Ditto, I don't think we'll need this linter, I don't expect this backend repo will rely heavily on translations